clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -2952,6 +2952,163 @@ check_exit_found(void)
     return true;
```

Comment:
> which is the next pc of the instr before the kernel trace. For traces without any kernel sequences, this would also be the immediately prior instruction,

A little confusing: is "this" the next pc, or the instr before?  Maybe can be clarified/reworded slightly.